src/app/angular2-multicomponent-form.component.ts

```diff
@@ -22,13 +21,35 @@ import { SidebarComponent } from './sidebar/sidebar.component';
 ])
 export class Angular2MulticomponentFormAppComponent {
   title = 'Angular2 Multi Component Dynamic Form demo';
+  baseHref = '/angular2-multicomponent-form'; 
```

Comment:
may be you can get it from the library: http://stackoverflow.com/questions/34535163/angular-2-router-no-base-href-set